src/org/zaproxy/zap/extension/ascanrules/TestSQLInjection.java

```diff
@@ -679,7 +692,7 @@ public void scan(HttpMessage msg, String param, String origParamValue) {
 			//String mResBodyNormal = getBaseMsg().getResponseBody().toString();
 			mResBodyNormalUnstripped = refreshedmessage.getResponseBody().toString();
 			mResBodyNormalStripped = this.stripOff(mResBodyNormalUnstripped, origParamValue);
-
+			
```

Comment:
Formatting changes should be submitted as a separate PR from code changes. https://github.com/zaproxy/zaproxy/wiki/DevGuidelines#formatting